src/main/fc/fc_tasks.c

```diff
@@ -366,13 +388,19 @@ cfTask_t cfTasks[TASK_COUNT] = {
         .staticPriority = TASK_PRIORITY_HIGH,
     },
 
-    [TASK_BATTERY] = {
-        .taskName = "BATTERY",
-        .taskFunc = taskUpdateBattery,
-        .desiredPeriod = TASK_PERIOD_HZ(50),        // 50 Hz
+    [TASK_BATTERY_ALERTS] = {
+        .taskName = "BATTERY_ALERTS",
+        .taskFunc = taskBatteryAlerts,
+        .desiredPeriod = TASK_PERIOD_HZ(5),        // 5 Hz
         .staticPriority = TASK_PRIORITY_MEDIUM,
     },
 
+    [TASK_BATTERY_SENSORS] = {
```

Comment:
previous comment updated, i pasted the same thing in twice.

You can see there is an extra 52 bytes of ram used and 40 bytes of flash on an F3 target.

we can remove this commit as you see fit, I'm happier with two tasks as you suggest as there is less code to maintain.